app/javascript/dashboard/routes/dashboard/settings/customRoles/component/CustomRolePaywall.vue

```diff
@@ -0,0 +1,94 @@
+<script setup>
+import { computed } from 'vue';
+import { useMapGetter } from 'dashboard/composables/store';
+import { useRouter } from 'dashboard/composables/route';
+import BasePaywallModal from 'dashboard/routes/dashboard/settings/components/BasePaywallModal.vue';
+import CustomRoleListItem from './CustomRoleTableBody.vue';
+
+const dummyCustomRolesData = [
+  {
+    name: 'All Permissions',
+    description: 'All permissions',
+    permissions: [
+      'conversation_manage',
+      'conversation_participating_manage',
+      'conversation_unassigned_manage',
+      'contact_manage',
+      'report_manage',
+      'knowledge_base_manage',
+    ],
+  },
+  {
+    name: 'Conversation Permissions',
+    description: 'Conversation permissions',
+    permissions: [
+      'conversation_manage',
+      'conversation_participating_manage',
+      'conversation_unassigned_manage',
+    ],
+  },
+  {
+    name: 'Contact Permissions',
+    description: 'Contact permissions',
+    permissions: ['contact_manage'],
+  },
+  {
+    name: 'Report Permissions',
+    description: 'Report permissions',
+    permissions: ['report_manage'],
+  },
+];
+
+const router = useRouter();
+
+const isOnChatwootCloud = useMapGetter('globalConfig/isOnChatwootCloud');
+
+const currentUser = useMapGetter('getCurrentUser');
+const currentAccountId = useMapGetter('getCurrentAccountId');
+
+const isSuperAdmin = computed(() => {
+  return currentUser.value.type === 'SuperAdmin';
+});
+const i18nKey = isOnChatwootCloud.value ? 'PAYWALL' : 'ENTERPRISE_PAYWALL';
```

Comment:
Should we use a computed property for better reactivity?
```suggestion
const i18nKey = computed(() => isOnChatwootCloud.value ? 'PAYWALL' : 'ENTERPRISE_PAYWALL');

```